### PR TITLE
LRQA-50723 and LRQA-68528 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRQA-50723
https://issues.liferay.com/browse/LRQA-68528

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/GDPR.macro [hunk 1]
```diff
@@ -91,32 +91,47 @@ definition {
...
-			if (isSet(anonymizeData)) {
-				AssertElementPresent(
-					key_tab = "Auto Anonymize Data",
-					locator1 = "NavTab#ACTIVE_TAB_LINK");
-			}
-			else {
-				AssertConfirm.assertConfirmationNoError(value1 = "All personal data associated with this user's applications has been deleted or anonymized. The user can now be deleted. Click OK to delete the user.");
-			}
+			Alert.viewSuccessMessage();
 		}
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/macros/GDPR.macro:126]: Great job splitting the macros! You might want to only include the `selectEntry` in the for statement. This current macro will delete them one by one as opposed to selecting multiple entries before deleting.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 2]
```diff
@@ -517,7 +556,7 @@ definition {
 			applications = "All Applications",
 			itemsCount = "1");
 
-		GDPR.deleteEntryCP();
+		GDPR.deleteSingleEntryCP();
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:559]: rowEntry variable needs to be set for the macro to know which entry it is selecting.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 3]
```diff
@@ -622,7 +661,7 @@ definition {
...
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:664]: same here, rowEntry variable needs to be set.

**Comment by patriciajeanperez** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**